data_steward/cdr_cleaner/cleaning_rules/generalize_zip_codes.py

```diff
@@ -0,0 +1,144 @@
+"""
+This cleaning rule generalizes all PPI concepts containing zip codes to their
+primary three digit representation, (e.g. 35400 → 354**).
+
+This cleaning rule is specific to the controlled tier.
+
+Original Issues: DC-1376
+"""
```

Comment:
We should mention that it will only generalize the zip code if the `observation_source_concept_id` is 1585250